dbrequests/database.py

```diff
@@ -35,23 +35,17 @@ def __init__(self, db_url=None, creds=None, sql_dir=None, connection_class=Defau
                 db = creds['db']
                 dialect = creds.get('dialect', 'mysql')
                 driver = creds.get('driver', 'pymysql')
-                self.db_url = '{}+{}://{}:{}@{}/{}'.format(dialect, driver, user, password, host, db)
+                self.db_url = '{}+{}://{}:{}@{}/{}'.format(
+                    dialect, driver, user, password, host, db)
             except:
                 raise ValueError('You must provide a db_url or proper creds.')
 
         self._escape_percentage = escape_percentage
         self._remove_comments = remove_comments
-        self._open = False
-        self.open(**kwargs)
+        self._engine = create_engine(self.db_url, **kwargs)
+        self._open = True
```

Comment:
whats the reason behind reverting this_

Reply:
I never planned to commit this after your last review. I think it just slipped through, and the reason for implementing it in the first place, has vanished; thanks to your last review.

Reply:
Not sure if we want to keep that.